src/components/FeatureViews/Notes/Notes.js

```diff
@@ -292,109 +323,249 @@ const Notes = ({ownerType, ownerID}) => {
-                            {numericDateString(note.timestamp)}
-                        </Typography>
-                        <div className={`actions ${classes.actionIcons}`}>
-                            <Delete className="icon"
-                                onClick={openDelete(note.id)} />
-                            <EditIcon className="icon"
-                                onClick={openExistingNote(note)} />
-                            <DoneIcon className="icon"
-                                onClick={toggleNoteField(note.id, "complete")}
-                                style={note.complete ? {"color": "#43B5D9"} : {}} />
-                        </div>
-                    </Paper>
+                    </DialogTitle>
+                    <DialogContent>
+                        <InputBase
+                            className="note-body"
+                            inputProps={{"aria-label": "naked"}}
+                            multiline
+                            onChange={handleBodyUpdate}
+                            placeholder="Body (required)"
+                            required
+                            rows={15}
+                            value={noteBody}
+                            variant="filled" />
+                    </DialogContent>
+                    <DialogActions>
+                        <Button
+                            onClick={hideWarning}
+                            variant="outlined">
+                            Cancel
+                        </Button>
+                        <Button
+                            color="primary"
+                            disabled={!noteBody || createResults.loading}
+                            onClick={saveNote}
+                            variant="outlined">
+                            {createResults.loading ? "Saving..." : "Save"}
+                        </Button>
+                        {
+                            createResults.error &&
+                            <span style={{"float": "right"}}>
+                                    Error while saving!
+                            </span>
+                        }
+                    </DialogActions>
+                </Dialog>
+                <Dialog
+                    aria-describedby="simple-modal-description"
+                    aria-labelledby="simple-modal-title"
+                    className="delete-popup"
+                    fullWidth
+                    maxWidth="xs"
+                    onClose={hideWarning}
+                    open={deleteID !== null}>
+                    <DialogTitle>
+                        Confirm Delete
+                    </DialogTitle>
+                    <DialogContent>
+                        Are you sure you want to delete {
+                            notes[deleteID] && notes[deleteID].title
+                                ? `"${notes[deleteID].title}"`
+                                : "this note"
```

Comment:
Revert this - notes can't be directly accessed by ID anymore, you have to use the `getNoteByID` function.